cmd/simulateEvol/simulateEvol.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/expandedTree"
+	"github.com/vertgenlab/gonomics/fasta"
+	"github.com/vertgenlab/gonomics/simulate"
+	"log"
+)
+
+func SimulateEvol(rootFastaFile string, treeFile string, simOutFile string, leafOutFile string) {
+	tree := expandedTree.ReadTree(treeFile, rootFastaFile)
+	var fastas []*fasta.Fasta
+	simulate.Simulate(rootFastaFile, tree)
+	nodes := expandedTree.GetTree(tree)
+
+	for i := 0; i < len(nodes); i++ {
+		fastas = append(fastas, nodes[i].Fasta)
+		if nodes[i].Fasta != nil {
```

Comment:
looks like there is something missing here

Reply:
... or maybe this if statement is no longer needed?